backend/package.json

```diff
@@ -9,14 +9,23 @@
     "nodemon": "^3.0.1"
   },
   "scripts": {
+    "test": "jest --coverage",
     "start": "nodemon server.js",
     "check": "prettier --check .",
     "format": "prettier --write .",
     "vercel-build": "echo yay"
   },
+  "jest": {
+    "transform": {
+      "^.+\\.js?$": "babel-jest"
+    }
+  },
   "devDependencies": {
-    "eslint": "^8.53.0",
+    "@babel/preset-env": "^7.23.2",
+    "babel-jest": "^29.7.0",
+    "eslint": "^8.52.0",
     "eslint-config-prettier": "^9.0.0",
-    "prettier": "^3.1.0"
+    "jest": "^29.7.0",
+    "prettier": "^3.0.3"
```

Comment:
Here can you kindly revert back to 3.1.0 it was updated recently the prettier version, might have missed during the merge conflict

Reply:
Resolved myself no worries :)